security/advancedtls/crl.go

```diff
@@ -192,25 +188,25 @@ func x509NameHash(r pkix.RDNSequence) string {
 	return fmt.Sprintf("%08x", fileHash)
 }
 
-// CheckRevocation checks the connection for revoked certificates based on RFC5280.
+// checkRevocation checks the connection for revoked certificates based on RFC5280.
```

Comment:
I don't see any usages of it inside the codebase, but it looks like it can be used as an embedded part of tls.Config. In case of OSS users use it that way, would it make sense just to mark it as deprecated (in favor of new CRL solution) and remove later?

Reply:
Can you link what you mean? I didn't see any usage in grpc-go or in google

Reply:
https://github.com/grpc/grpc-go/blob/0baa668e3dcc443261c266d8914d3321d20d2eaa/security/advancedtls/crl_test.go#L739

Reply:
For me, I think we should unexport - we don't want anyone depending on this. If someone is using it in this way currently, they can stay on a version of the advancedTLS package that has it until they move off.
It's not really something to deprecate, rather it's an internal API that was written with a capital letter

@dfawley what do you think here?